service_manifest.yml

```diff
@@ -2,8 +2,8 @@ name: Extract
 version: $SERVICE_TAG
 description: This service extracts embedded files from file containers (like ZIP, RAR, 7z, ...).
 
-accepts: (archive|executable|java)/.*|code/vbe|code/html|document/email|document/pdf|document/office/passwordprotected|android/apk|ios/ipa
-rejects: empty|metadata/.*
+accepts: (archive|executable|java)/.*|code/vbe|code/html|document/email|document/pdf|document/office/passwordprotected|android/apk|ios/ipa|metadata/sysmon
+rejects: empty
```

Comment:
should we maintain the fact that it rejects all other metadata?  Also, not sure if the `accepts` or the `rejects` takes precedence

Reply:
rejects takes precedence. As it is right now if we block file types starting with metadata, it will also block all sysmon logs

Reply:
An option could be changing the type in identify.py to just "sysmon/" but I'm not sure if that's a good idea.

Reply:
I definitely think that would be worth discussing. At this stage, it is a `sysmon` file more than it is a `metadata/sysmon` file